Packages/NetworkKit/Sources/NetworkKit/MockNetwork.swift

```diff
@@ -32,6 +31,16 @@ public class MockNetwork: Networking {
         .eraseToAnyPublisher()
     }
 
+    public func performRequest<E>(endpoint: E) async throws -> E.DataType where E : Endpoint {
```

Comment:
I noticed the protocol I copied over was missing the async function from its definition so I added it, and then added the relevant code to MockNetwork